.github/workflows/post-merge-validation.yml

```
@@ -0,0 +1,283 @@
+    timeout-minutes: 10
+    
+    steps:
+      - name: 📥 Checkout code
+        uses: actions/checkout@v4
+
+      - name: 🔍 Wait for Vercel Deployment
+        uses: ./.github/actions/wait-for-vercel
+        id: deployment
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          timeout: 600
```

actionlint (1.7.7)

input "timeout" is not defined in action "Wait for Vercel Deployment" defined at "./.github/actions/wait-for-vercel". available inputs are "check-interval", "github-token", "timeout-minutes"

(action)